OpenRA.Game/Exts.cs

```diff
@@ -372,13 +372,17 @@ public static T[] MakeArray<T>(int count, Func<int, T> f)
 		{
 			var result = new T[width, height];
 			for (var i = 0; i < width; i++)
+			{
 				for (var j = 0; j < height; j++)
-					// Workaround for broken ternary operators in certain versions of mono (3.10 and  
-					// certain versions of the 3.8 series): https://bugzilla.xamarin.com/show_bug.cgi?id=23319
+				{
+					// Workaround for broken ternary operators in certain versions of mono (3.10 and certain versions of the 3.8 series): https://bugzilla.xamarin.com/show_bug.cgi?id=23319
```

Comment:
Didn't know how else to solve this StyleCop nit. Let's hope this does not have to stay forever here.